packages/rs-drive-abci/Cargo.toml

```diff
@@ -74,6 +72,8 @@ tokio = { version = "1", features = [
 ] }
 tokio-util = { version = "0.7.8" }
 derive_more = "0.99.17"
+metrics = "0.22.1"
```

Comment:
For those who a curious about this change:

a new dependency on json-schema-faker-rs introduced indirect dependency on a version of hashbrown library above X, when older metrics version was set to use hashbrown below X, in recent version they bumped their dependencies as well and conflict is resolved